app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
I'm curious what it's used for in Hyrax. In general, yes I'd say that dams-support address is probably fine, but it would be nice if we knew a bit more about the context.